### PR TITLE
Add input data into localstorage

### PR DESCRIPTION
**Pull request by jerryowusu**
## Local Storage

In this milestone we:

- Added a function to store input data from local storage

### CODE REVIEW

File: js/validation.js [hunk 1]
```diff
@@ -17,3 +19,21 @@ form.addEventListener('submit', (event) => {
...
+  localStorage.setItem('data', JSON.stringify(formData));
+});
+let storedData = localStorage.getItem('data'); storedData = JSON.parse(storedData);
+window.onload = () => {
+  if (storedData) {
+    nameInput.value = storedData.Name;
+    email.value = storedData.Email;
+    message.value = storedData.Message;
+  }
+};
```
Comment (rotimiazeez) [inline js/validation.js:39]: - [x] Please note that on page reload the information entered in the contact form and not submitted should be retained and not lost 😎😎. Kindly make sure you look into this😉

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team👋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/validation.js (hunk 1 above)
Comment (jerryowusu) [reply js/validation.js:39]: Please I have tried and tested several times and it is working. My partner has also tried it and it is working fine.
If it's still not working, kindly try switching to the local-storage branch in your code editor. Thanks

**Review by oloomoses — APPROVED** [review]
Hi Team, 
Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**